lib/fastlane/actions/reset_simulator.rb

```diff
@@ -0,0 +1,76 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      RESET_SIMULATOR_CUSTOM_VALUE = :RESET_SIMULATOR_CUSTOM_VALUE
+    end
+
+    class ResetSimulatorAction < Action
+      def self.run(params)
+        require 'json'
+
+        unless min_xcode7?
+          Helper.log.error "Xcode version 7 or higher is required!".red
+          return
+        end
+
+        json = JSON.parse(Actions.sh_no_action("xcrun simctl list --json devices", log: false))
+        json["devices"].each do |os, devices|
+          devices.each do |device|
+            next unless params[:device] == "all" ||
```

Comment:
Could you split that up into multiple lines somehow to make it more clear on what we try to archive here?